rllib/agents/ppo/ppo_tf_policy.py

```diff
@@ -349,6 +374,16 @@ def setup_mixins(policy: Policy, obs_space: gym.spaces.Space,
     LearningRateSchedule.__init__(policy, config["lr"], config["lr_schedule"])
 
 
+def view_requirements_fn(policy):
+    # Adds the input-dict used in postprocessing to the dict of view-reqs.
+    # This is for value calculation at the very end of a trajectory
+    # (abs_pos=-1). It's only used if the trajectory is not finished yet and we
+    # have to rely on the last value function output as a reward estimation.
+    return {
+        "_value_input_dict": ViewRequirement(is_input_dict=True, abs_pos=-1)
```

Comment:
Wondering if there are better names for this
```
# Can specify either abs_index or shift
abs_index=-1
shift=x
```

I think the notion of "index" is clearer in Python (-1 index means end). Also, we get to keep shift.

Reply:
Fair enough, I'll rename these two:
abs_pos -> index
data_rel_pos -> shift

Reply:
On the input_dict:
It's not necessary for this very PR, but as this is a preparatory PR (to make the attention PR smaller) I decided to already add this here. The attention net PR needs this feature to be able to not have to boiler-plate/hardcode the attention logic inside e.g. PPO's postprocessing fn (this function should not have to worry about the model being an RNN or attention net, it should not need to know).